### PR TITLE
added orca detection code

### PR DESCRIPTION
**Pull request by ZER-0-NE**
This commit adds 4 scripts for the issue #22  namely,
- `orcacnn_detection.py`: This contains my network configuration for the CNN.
- `predict.py`: This script predicts which images are orcas by using the saved model weights and copies the detected orca calls in a new folder which assists in determining start and end time of orcas.
- `report.py`: This script prints a [sklearn classification report](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.classification_report.html). @yosoyjay should I add those 70 pos and neg samples?
- `determine_calls.py`: Determine the start and end time of orca calls.

The `Trained Notebook` contains one of my trained notebook which achieved 95% test accuracy.

I'll be updating the readme in a few days.

### CODE REVIEW

**Comment by yosoyjay** [comment]
@ZER-0-NE Let's kept this repo focused on code and documentation.  We can archive that data on Google Cloud.
